src/includes/set-context/python.mdx

```diff
@@ -1,10 +1,9 @@
 ```python
-from sentry_sdk import configure_scope
+from sentry_sdk import set_context
```

Comment:
These `from sentry_sdk import *` examples look non-idiomatic to me. I expect most users to be writing `import sentry_sdk` and `sentry_sdk.FOO()` instead of bringing parts of the SDK into the global namespace.